docs/output.md

```diff
@@ -16,11 +16,11 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 
 [FRED2](https://github.com/FRED-2) is used to perform the prediction of Epitopes on the given data, independent of the chosen `tools` to perform the prediction.
 
-**Output directory: `results/`**
+**Output directory: `predictions/`**
 
-* `prediction_report.json`
+* `${input_base_name}_prediction_report.json`
```

Comment:
This is inconsistent with the notation used for `[tool].[version].supported_alleles.txt`. Seems every pipeline does this differently. Use either `[...]` or `${...}` in the whole `output.md`.